src/main/java/org/pentaho/di/profiling/datacleaner/ModelerHelper.java

```diff
@@ -144,8 +144,10 @@ public static int launchDataCleaner(DataCleanerSpoonConfiguration dataCleanerSpo
 
         try {
             final String dcInstallationPath = dataCleanerSpoonConfiguration.getDataCleanerInstallationFolderPath()
-                    + "/DataCleaner.jar";
-            StringBuilder classPathBuilder;
+                    + "/DataCleaner.jar" + ";" + dataCleanerSpoonConfiguration.getDataCleanerInstallationFolderPath()
+                    + "/modules/*" + ";" + dataCleanerSpoonConfiguration.getDataCleanerInstallationFolderPath()
```

Comment:
Maybe not big enough to fix now, but it is getting a bit messy. I'd consider something like this:

```java
    private static String asPath(String... filenames) {
        return Stream.of(filenames).map(f -> getDataCleanerInstallationFolderPath() + "/" + f).collect(
                Collectors.joining(";"));
    }
    
   ...
      final String dcInstallationPath = asPath("DataCleaner.jar", "modules/", "lib/*");
```
("filenames" as the argument name is maybe a bit wrong, being both a filename and paths, but I could't really conjure up anything better :))

Maybe not much gained now, but changing it later is a little simpler.